README.md

```diff
@@ -218,6 +219,7 @@ Here are all the samples which are deployed on Microsoft Azure and you can try i
 [messageswithrsc#csharp]:samples/bot-receive-channel-messages-withRSC/csharp
 [messageswithrsc#js]:samples/bot-receive-channel-messages-withRSC/nodejs
 [app-in-meeting#cs]:samples/app-in-meeting/csharp
+[qrappinstallation#nodejs]:samples/QR-app-installation/nodejs
```

Comment:
Please make the same changes (C# PR) for folder name and image in readme file.

Reply:
done